Lib/idlelib/idle_test/test_textview.py

```diff
@@ -129,8 +129,11 @@ def _command():
 
         self.assertEqual(self.called, True)
         self.assertEqual(self.view.title(), 'TITLE_FILE')
-        self.assertEqual(self.view.textView.get('1.0', '1.end'),
-                         "'''Test idlelib.textview.")
+        with open(__file__) as f:
+            self.assertEqual(self.view.textView.get('1.0', '1.end'),
+                             f.readline().strip())
+            self.assertEqual(self.view.textView.get('2.0', '2.end'),
+                             f.readline().strip())
```

Comment:
I like making this future proof.  However, since the 2nd line is blank and Texts have a default extra \n at the end, I believe this would pass even if the textview only displayed one line.  So I added an intermediate `f.readline()` and changed 2s to 3s.